### DISCUSSION TITLE
Welcome to crypto_price_prediction Discussions!

### DISCUSSION THREAD

**Discussion by paulcodrea** [paulcodrea/dissertation#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Difab91** [comment]
Hello, I'm working on a project, I want to retrieve twitter posts on specific dates via the tweeter api. unfortunately I get a problem authorizing access. I contact you because I found your project and look at your scrap code similar to mine, I'm even inspired. My question is as follows: have you managed to scrape your information with the free access level of the api or the basic access (paid)? my problem may be due to the fact that I have the free version. If not, do you know anything about the Twitter API to help me solve my error (Forbidden: 403 Forbidden
453 - You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level)

sincerely
